packages/typescript-estree/tests/lib/convert.test.ts

```diff
@@ -437,10 +436,10 @@ describe('convert', () => {
     it('allows writing to the deprecated getter property as a new enumerable value', () => {
       const tsMappedType = getEsTsMappedType();
 
-      // eslint-disable-next-line deprecation/deprecation, @typescript-eslint/no-unused-expressions
+      // eslint-disable-next-line deprecation/deprecation
       tsMappedType.typeParameter = undefined!;
 
-      // eslint-disable-next-line deprecation/deprecation, @typescript-eslint/no-unused-expressions
+      // eslint-disable-next-line deprecation/deprecation
```

Comment:
too many `@typescript-eslint/no-unused-expressions` were accidentally added in https://github.com/typescript-eslint/typescript-eslint/pull/9079/commits/aa5525f0dfd15265ebd4a364dcc5aca12bfc676d